### PR TITLE
Boom wires have rudimentary logging.

### PR DESCRIPTION
**Pull request by harmonyn**
Fixes https://github.com/tgstation/-tg-station/issues/4079, partially.

I want to find some way to get fingerprints or location from the user but haven't figured out how yet.

### CODE REVIEW

**Comment by coiax** [comment]
Wouldn't fingerprints be generated when they open the wire panel

**Comment by bgobandit** [comment]
possibly but the arguments here are the bomb and the wire datum

File: code/datums/wires/syndicatebomb.dm [hunk 1]
```diff
@@ -74,9 +76,16 @@
...
 			if(!mend && B.active)
 				holder.visible_message("<span class='notice'>\icon[B] The timer stops! The bomb has been defused!</span>")
 				B.active = FALSE
 				B.defused = TRUE
 				B.update_icon()
+
+/datum/wires/syndicatebomb/proc/tell_admins(obj/machinery/syndicatebomb/B)
+	if(istype(B, /obj/machinery/syndicatebomb/training))
+		return
+	log_game("A [B.name] was detonated via boom wire at <A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[B.x];Y=[B.y];Z=[B.z]'> (JMP)</a>.")
```
Comment (optimumtact) [inline code/datums/wires/syndicatebomb.dm]: there's no point putting the href jmp here I dont' think, just print the x, y z

Comment (optimumtact) [inline code/datums/wires/syndicatebomb.dm]: bandit pls

Comment (bgobandit) [inline code/datums/wires/syndicatebomb.dm]: as an admin I'd rather have it

Comment (optimumtact) [inline code/datums/wires/syndicatebomb.dm]: there's no point including the HREF, the log doesn't go to the chat window.

I'm saying print the X Y and Z, and remove the href jump

**Comment by AnturK** [comment]
@bgobandit Please

**Pull request closed without merging**